esp/esdllib/esdl_script.cpp

```diff
@@ -29,6 +29,61 @@
 #include <fxpp/FragmentedXmlPullParser.hpp>
 using namespace xpp;
 
+class CEsdlScriptContext : public CInterfaceOf<IEsdlScriptContext>
+{
+public: // ISectionalXmlDocModel
+    virtual IXpathContext* createXpathContext(IXpathContext* parent, const char* section, bool strictParameterDeclaration) override { return docModel->createXpathContext(parent, section, strictParameterDeclaration); }
+    virtual IXpathContext* getCopiedSectionXpathContext(IXpathContext* parent, const char* tgtSection, const char* srcSection, bool strictParameterDeclaration) override { return docModel->getCopiedSectionXpathContext(parent, tgtSection, srcSection, strictParameterDeclaration); }
+    virtual void* queryUserData() override { return this; }
```

Comment:
This optimization is what lead me to investigate whether this userData was being used anymore.